hydromt/models/model_lumped.py

```diff
@@ -153,56 +183,67 @@ def __init__(
             data_libs=data_libs,
             logger=logger,
         )
-
-    def read(
```

Comment:
should read indeed be deleted?

Reply:
idem for write

Reply:
I put read and write back and the region property to. Indeed to enthusiast removing these -> confusion with the write_response_units in the mixedin class. Also,it seems that the addition here is minimal, only a different list is passed on to the super function.